assets/bindata.go

```diff
@@ -1,6 +1,5 @@
-// Code generated by go-bindata. (@generated) DO NOT EDIT.
-
-// Package assets generated by go-bindata.// sources:
+// Code generated for package assets by go-bindata DO NOT EDIT. (@generated)
+// sources:
```

Comment:
What version of `go-bindata` are you using?
```
$ go-bindata --version
go-bindata 3.1.3 (Go runtime go1.15.8).
Copyright (c) 2010-2013, Jim Teeuwen.
```

Reply:
@vkareh 

 go-bindata 3.1.2 (Go runtime go1.16.9).
Copyright (c) 2010-2013, Jim Teeuwen.

I ran make generate

Reply:
okay, I updated with:
```
$ GO111MODULE=off go get -u github.com/go-bindata/go-bindata/...
$ go-bindata --version
go-bindata 3.1.3 (Go runtime go1.16.8).
Copyright (c) 2010-2013, Jim Teeuwen.
```
it still re-generates this file differently - maybe run that command (which the Makefile also does if you don't already have the binary) to update to 3.1.3